portfolio/components/Navbar.jsx

```diff
@@ -0,0 +1,104 @@
+'use client'
+import Link from 'next/link'
+import Image from 'next/image';
+import { useState } from 'react'
+import { logo, sunIcon, moonIcon, hamburger, cross } from '../assets';
```

Comment:
@ConnorHoare Be sure to separate the internal from the external imports by placing external imports at the top of the file, leaving a space between them. 

```
// External imports
import Link from 'next/link';
import Image from 'next/image';
import { useState } from 'react';
import { useTheme } from 'next-themes';

// Internal imports
import { logo, sunIcon, moonIcon, hamburger, cross } from '../assets';

```